packages/babel-plugin-emotion/src/utils/transform-expression-with-styles.js

```diff
@@ -53,13 +53,6 @@ export let transformExpressionWithStyles = ({
       arg => arg.type !== 'SpreadElement'
     )
 
-    if (canAppendStrings && shouldLabel) {
```

Comment:
Moving this further down allows us to remove labels from production styles when we are able to compute "opaqueCss" object with Babel. A followup PR can also wrap appending label with NODE_ENV check to avoid the same thing for the runtime serialization case.